Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -25,7 +27,8 @@ open class TopBannerViewController: UIViewController {
     
     lazy var informationStackView = UIStackView(orientation: .vertical, autoLayout: true)
     
-    lazy var instructionsBannerView: InstructionsBannerView = {
+    /// A banner view that contains the current step instructions and responds to tap and swipe gestures.
+    public lazy var instructionsBannerView: InstructionsBannerView = {
```

Comment:
Shouldn't this view be `public private(set)` instead? Otherwise, if modified, this could break the layout. At least it should have probably a `didSet` handler which does the layout setup.